schemas/biomarkers.json

```diff
@@ -0,0 +1,481 @@
+        "regex": "#/regex/submitter_id",
+        "script": ""
+      }
+    },
+    {
+      "name": "submitter_primary_diagnosis_id",
+      "valueType": "string",
+      "description": "If the biomarker test was done at the time of primary diagnosis, then indicate the associated submitter_primary_diagnosis_id here.",
+      "meta": {
+        "displayName": "Submitter Primary Diagnosis ID",
+        "primaryId": true
+      },
+      "restrictions": {
+        "regex": "#/regex/submitter_id",
+        "script": ""
+      }
+    },
+    {
+      "name": "submitter_treatment_id",
+      "valueType": "string",
+      "description": "If the biomarker test was done at the initiation of a specific treatment regimen, indicate the associated submitter_treatment_id here.",
+      "meta": {
+        "primaryId": true,
+        "displayName": "Submitter Treatment ID"
+      },
+      "restrictions": {
+        "regex": "#/regex/submitter_id",
+        "script": ""
+      }
+    },
+    {
+      "description": "If the biomarker test was done during a follow up event, then indicate the associated submitter_follow_up_id here.",
+      "name": "submitter_follow_up_id",
+      "valueType": "string",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "displayName": "Submitter Follow-Up ID"
+      },
+      "restrictions": {
+        "regex": "#/regex/submitter_id",
+        "script": ""
+      }
+    },
+    {
+      "description": "If the biomarker test was not associated with a specific followup, primary diagnosis or treatment event, then indicate the interval of time since primary diagnosis that the biomarker test was performced at, in days.",
+      "name": "test_interval",
+      "valueType": "integer",
+      "restrictions": {
+        "range": {
+           "exclusiveMin": 0,
+           "script": ""
+        }
+      },
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "units": "days",
+        "displayName": "Interval Of Test",
+        "notes": "This field is required if the biomarker test is not associated with a primary diagnosis, treatment or follow up event. The associated primary diagnosis is used as the reference point for this interval. To calculate this, find the number of days since the date of primary diagnosis."
+      }
+    },
+    {
+      "description": "Indicate the level of carbohydrate antigen 19-9. Carbohydrate antigen 19-9 testing is useful to monitor the response to treatment in pancreatic cancer patients. (Reference: LOINC: 24108-3)",
+      "name": "ca19-9_level",
+      "valueType": "integer",
+      "restrictions": {
+        "range": {
+           "exclusiveMin": 0
+        }
+      },
+      "meta": {
+        "displayName": "CA19-9 Level"
+      }
+    },
+    {
+      "description": "Indicate the quantitative measurement of the amount of CRP, an inflammatory marker, in the blood in mg/L. (Reference: NCIt C64548)",
```

Comment:
should CRP be spelled out `...  amount of  C-reactive protein (CRP)...`?

Reply:
Updated to `C-reactive protein (CRP)`